.expo/packager-info.json

```diff
@@ -1,5 +1,6 @@
 {
   "expoServerPort": null,
   "packagerPort": null,
-  "packagerPid": null
+  "packagerPid": null,
```

Comment:
Can you remove .expo files from the project?

Reply:
I have removed the expo files